src/scripts/main.js

```diff
@@ -1,3 +1,381 @@
 'use strict';
 
-// write code here
+const tableHeaders = [...document.querySelectorAll('thead th')];
```

Comment:
There is a syntax error in the way `document.querySelectorAll` is used. The square brackets should not be there. It should be `document.querySelectorAll('thead th')` without the brackets.